app/views/tracks/exercises/show/_action_box_join_exercism.html.haml

```diff
@@ -10,7 +10,7 @@
     Sign up to Exercism to learn and master
     = link_to track.title, track_path(track)
     with
-    - if user_track.num_concepts.positive?
+    - if @track.course? && user_track.num_concepts.positive?
```

Comment:
For all of these in this PR, I think we can just do `@track.course?` and not bother with `user_track.num_concepts.positive?`. `configlet` shouldn't let tracks be courses without concepts, etc.

Reply:
> configlet shouldn't let tracks be courses without concepts, etc.

This isn't yet the case, but I can fix that easily enough.